OracleJava/README.md

```diff
@@ -1,21 +1,21 @@
 Oracle Java on Docker
```

Comment:
Please replace "on Docker" with "in a container"

Reply:
Done